release notes/v0.52.0.md

```diff
@@ -0,0 +1,141 @@
+- [browser#1349](https://github.com/grafana/xk6-browser/pull/1349), [browser#1354](https://github.com/grafana/xk6-browser/pull/1354) Fixes `Request` mappings.
+- [browser#1334](https://github.com/grafana/xk6-browser/pull/1334) Fixes an issue where clicking on a link that opens a new tab never navigates to the href link.
+- [browser#1318](https://github.com/grafana/xk6-browser/pull/1318) Fixes the `Mouse.move` to correctly dispatch a `down` event.
+- [browser#1301](https://github.com/grafana/xk6-browser/pull/1301) Fixes an error that occurs when working with a second tab and navigating to a URL.
+- [browser#1387](https://github.com/grafana/xk6-browser/pull/1387) Fixes a panic when the new document or request is missing.
+
+## Maintenance and internal improvements
+
+- [#3752](https://github.com/grafana/k6/pull/3752), [#3767](https://github.com/grafana/k6/pull/3767), [#3770](https://github.com/grafana/k6/pull/3770), [#3780](https://github.com/grafana/k6/pull/3780), [#3782](https://github.com/grafana/k6/pull/3782), [#3795](https://github.com/grafana/k6/pull/3795) updates several dependencies.
+- [#3786](https://github.com/grafana/k6/pull/3786), [#3787](https://github.com/grafana/k6/pull/3787), [#3788](https://github.com/grafana/k6/pull/3788), [#3789](https://github.com/grafana/k6/pull/3789), [#3803](https://github.com/grafana/k6/pull/3803) updates experimental modules.
+- [#3749](https://github.com/grafana/k6/pull/3749) stops printing `goja` stack traces on panics, which has been empty since a while.
+- [#3760](https://github.com/grafana/k6/pull/3760) fixes race conditions in ramping-vus tests.
+- [#3769](https://github.com/grafana/k6/pull/3769) removes a linter setting no longer used.
+- [#3800](https://github.com/grafana/k6/pull/3800) adds test coverage for outputting the archive to stdout.
+
+- [browser#1298](https://github.com/grafana/xk6-browser/pull/1298) Splits browser mappings into separate files for better organization and maintainability.
+- [browser#1321](https://github.com/grafana/xk6-browser/pull/1321) Adds test helpers for async VU execution to reduce the boilerplate in the tests.
+- [browser#1357](https://github.com/grafana/xk6-browser/pull/1357) Updates `Group` and `Tag` usage for the latest k6.
+- [browser#1361](https://github.com/grafana/xk6-browser/pull/1361) Precalculates the browser version information to prevent I/O calls when the `version` and `userAgent` methods are called. This change allows the browser module to expose these methods as a sync API to be consistent with the Playwright's API.
+- [browser#1377](https://github.com/grafana/xk6-browser/pull/1377) Uses the Goja fork called Sobek in the browser module.
+- [browser#1373](https://github.com/grafana/xk6-browser/pull/1373) Provides sync and async APIs within the browser module for making the migration to the async API easier.
```

Comment:
```suggestion
- [browser#1298](https://github.com/grafana/xk6-browser/pull/1298) splits browser mappings into separate files for better organization and maintainability.
- [browser#1321](https://github.com/grafana/xk6-browser/pull/1321) adds test helpers for async VU execution to reduce the boilerplate in the tests.
- [browser#1357](https://github.com/grafana/xk6-browser/pull/1357) updates `Group` and `Tag` usage for the latest k6.
- [browser#1361](https://github.com/grafana/xk6-browser/pull/1361) precalculates the browser version information to prevent I/O calls when the `version` and `userAgent` methods are called. This change allows the browser module to expose these methods as a sync API to be consistent with the Playwright's API.
- [browser#1377](https://github.com/grafana/xk6-browser/pull/1377) uses the Goja fork called Sobek in the browser module.
- [browser#1373](https://github.com/grafana/xk6-browser/pull/1373) provides sync and async APIs within the browser module for making the migration to the async API easier.
```